### PR TITLE
10 Mejorando los selectores

### PR DESCRIPTION
**Pull request by JorgeVQA**
Added several new css selectors, also used couple of "classes", names, types among others.

### CODE REVIEW

File: cypress/page/menu-content.page.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ class MenuContentPage {
 
   constructor() {
     this.menuContentPageURL = "http://automationpractice.com/"
-    this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";  
+    this.tShirtMenu = ".sf-menu > :nth-child(3) > a";
```
Comment (holgiosalos) [inline cypress/page/menu-content.page.ts]: When the selector depends on the order of any element, we can have the risk of having a flaky test. Could you think of a way to not use the `:nth-child(3)` function?

File: cypress/page/payment-step.page.ts [hunk 2]
```diff
@@ -4,9 +4,9 @@ class PaymentStepPage {
   private orderConfirmationMessage: string
   
   constructor() {
-    this.paymentMethod = "div#HOOK_PAYMENT div:nth-child(1) > div > p > a";
-    this.confirmOrderBtnPayment = "p#cart_navigation span";
-    this.orderConfirmationMessage = "#center_column > div > p > strong";
+    this.paymentMethod = ".bankwire";
+    this.confirmOrderBtnPayment = "#cart_navigation > .button > span";
```
Comment (holgiosalos) [inline cypress/page/payment-step.page.ts]: I would say that `#cart_navigation span` will be enough

File: cypress/page/shopping-cart.page.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ class ShoppingCartPage {
   private proceedToCheckoutBtnCart: string
   
   constructor() {
-    this.proceedToCheckoutBtnCart = ".cart_navigation span";
+    this.proceedToCheckoutBtnCart = ".cart_navigation > .button > span";
```
Comment (holgiosalos) [inline cypress/page/shopping-cart.page.ts]: the previous one was not bad at all

File: cypress/page/products-list.page.ts [hunk 4]
```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
   private proceedToCheckoutBtnPopUp: string
   
   constructor() {
-    this.productToAdd = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.proceedToCheckoutBtnPopUp = "[style*=\"display: block;\"] .button-container > a";
+    this.productToAdd = ".ajax_add_to_cart_button > span";
+    this.proceedToCheckoutBtnPopUp = ".button-container > .button-medium > span";
```
Comment (holgiosalos) [inline cypress/page/products-list.page.ts:7]: maybe `a[title="Proceed to checkout"]` or a contains can work better

File: cypress/page/menu-content.page.ts (hunk 1 above)
Comment (JorgeVQA) [reply cypress/page/menu-content.page.ts]: I've found a new css selector without the **:nth-child( )** one.

File: cypress/page/payment-step.page.ts (hunk 2 above)
Comment (JorgeVQA) [reply cypress/page/payment-step.page.ts]: Thanks, fixed.

File: cypress/page/shopping-cart.page.ts (hunk 3 above)
Comment (JorgeVQA) [reply cypress/page/shopping-cart.page.ts]: Fixed as well

File: cypress/page/products-list.page.ts (hunk 4 above)
Comment (JorgeVQA) [reply cypress/page/products-list.page.ts:7]: Fixed and used the new one.

**Review by manuelq12 — APPROVED** [review]
LGTM

**Pull request merged**